parquet/src/test/java/org/apache/iceberg/parquet/TestCDHParquetStatistics.java

```diff
@@ -39,6 +39,6 @@ public void testCDHParquetStatistcs() {
     when(cdhBinaryColumnStats.getMaxBytes()).thenReturn(null);
     when(cdhBinaryColumnStats.getMinBytes()).thenReturn(null);
     when(cdhBinaryColumnStats.getNumNulls()).thenReturn(0L);
-    Assert.assertTrue(ParquetMetricsRowGroupFilter.minMaxUndefined(cdhBinaryColumnStats));
+    Assertions.assertTrue(ParquetMetricsRowGroupFilter.minMaxUndefined(cdhBinaryColumnStats));
```

Comment:
should be `Assertions.assertThat(...).isTrue()`

Reply:
It was using Junit Assertions, changed to AssertJ style